core-infrastructure/src/db/Core.Database/Scripts/001-CreateComparatorSetTable.sql

```diff
@@ -0,0 +1,16 @@
+IF NOT EXISTS(SELECT *
+               FROM INFORMATION_SCHEMA.TABLES
+               WHERE table_name = 'ComparatorSet')
+    BEGIN
+        CREATE TABLE dbo.ComparatorSet
+        (
+            RunType  nvarchar(50)  NOT NULL,
```

Comment:
More of a general question about the columns: a lot are `nvarchar(50)` with only a few exceptions, is this because we don't know the range of values that _should_ be in there and can't constrain things further?

Reply:
@PsypherPunk exactly that. Didn't want to constrain things too much at this point.